test/unit/rope_out_of_memory.c

```diff
@@ -0,0 +1,87 @@
+#include "unit.h"
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
```

Comment:
Including stdio and stdlib headers seem to be redundant

Reply:
fix